eclipse-collections-code-generator/src/main/resources/impl/bag/mutable/unmodifiablePrimitiveBag.stg

```diff
@@ -110,6 +112,14 @@ public class Unmodifiable<name>Bag
         return this.getMutable<name>Bag().selectByOccurrences(predicate);
     }
 
+    @Override
+    public Mutable<name>Set selectUnique()
```

Comment:
Similar to Synchronized, you can delegate it to the underlying MutableBag, it changes it to one line implementation.

`return this.getMutable<name>Bag().selectUnique();`